realm/src/main/java/io/realm/internal/TableQuery.java

```diff
@@ -402,70 +402,70 @@ public TableQuery notEqualTo(long columnIndex[], String value){
     }
     protected native void nativeNotEqual(long nativeQueryPtr, long columnIndex[], String value, boolean caseSensitive);
 
-    public TableQuery beginsWith(long columnIndex, String value, boolean caseSensitive){
+    public TableQuery beginsWith(long columnIndices[], String value, boolean caseSensitive) {
```

Comment:
I don't think we should remove the option to just provide one sort parameter in a simple fashion - then better provide both options. (goes for all methods taking columnIndices[])